### ISSUE TITLE
Post bot stats to master process

### ISSUE THREAD

**Issue by acollierr17** [suggestionsbot/suggestions-bot-js#23]
So I've been experimenting in the past with this feature but to no avail. I would like to get stats such as guild count from each cluster and have it sent to master so I can build an aggregate count. Eventually, I'll have this aggregate data either saved to Redis or post to an external API to be eventually sent to bot lists.

For the sharding manager, I am using https://github.com/DevYukine/Kurasuta. I've worked with the README and code the best I can, but I still struggled to find a good way to actually achieve this.

Here's a quick and dirty rundown of what I want to do:

1. Have each cluster send a message to master with its `cluster_id` and `guild_count`
2. Once the master process has received all messages from all clusters, get an aggregate count for `guild_count`
3. Take the aggregate and do whatever I want with it (ex. post to Redis or send to an API)

I've chatted with a few friends on how to approach this in the best. The ideas helped, but trying to execute with not the best documentation provided in Kurasuta only caused me to struggle. I'm hoping someone can shine some light or provide some insight here as I'd like to ship out this feature sooner than later so I can get accurate data posted to bot lists.

**Comment by Giuliopime** [comment]
I usually just gather stats from my master process and save it in the db, every like 1 or more hours.

**Comment by acollierr17** [comment]
> I usually just gather stats from my master process and save it in the db, every like 1 or more hours.

Yeah, I've actually thought of doing that as well. I'm just trying to figure out the best way to do it with Kurasuta, which has been the main issue for me.

**Comment by acollierr17** [comment]
So after digging through the Kurasuta codebase a bit more (every time I look through, I tend to find something new), I found this method which is based on discord.js' v13 `ShardingManager#broacast` function:
https://github.com/DevYukine/Kurasuta/blob/1c7118641f4c8bedd0af91163013737aa54dd3c8/src/Sharding/ShardingManager.ts#L148-L150

This function here solves all the problems I encountered above. I'm still using discord.js v12, so the `broadcast` function is news to me, hence me not noticing it in the first place. I'll have this implemented in the next update.
